js/sykepengesoknad/data/ettersending/ettersendingNav.js

```diff
@@ -53,6 +58,14 @@ export default (state = initiellState, action = {}) => {
                 sendt: false,
             };
         }
+        case ETTERSEND_SOKNAD_NAV_NULLSTILL: {
```

Comment:
Kan du ikke bare returnere initiellState? 

Generelt syns jeg det er bedre hvis action-typene beskriver ting som _har_ skjedd i stedet for ting som _skal_ skje :)

Reply:
jo, det kan jeg :)

Kan se om jeg klarer å komme på nye navn.